packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -101,7 +103,8 @@ export class ForceApexLogGetExecutor extends SfdxCommandletExecutor<
       const localUTCDate = new Date(response.data.startTime);
       const date = getYYYYMMddHHmmssDateFormat(localUTCDate);
       const logPath = path.join(logDir, `${response.data.id}_${date}.log`);
-      fs.writeFileSync(logPath, resultJson.result.log);
+      // this needs to be result[0] for this to work correctly but idk if that used to be necessary
+      fs.writeFileSync(logPath, resultJson.result[0].log);
```

Comment:
will remove this, related to a bug we discovered

Reply:
@AnanyaJha is this the fix for the `undefined` error we've been discussing ?

Reply:
Yeah that's the fix, but I'm still looking into what might have caused it